### ISSUE TITLE
Strange sample size from orix.sampling.get_sample_fundamental() for some resolutions

### ISSUE THREAD

**Issue by hakonanes** [pyxem/orix#190]
After #177, I'm getting an unexpected decrease in sample size with increasing characteristic distance ("resolution"):

```python
>>> import matplotlib.pyplot as plt
>>> import numpy as np
>>> from orix import quaternion, sampling
>>> pg432 = quaternion.symmetry.O
>>> n = 10
>>> sampling_resolution = np.linspace(1.5, 2, num=n)
>>> sampling_size = np.zeros(n)
>>> for i in range(n):
...     sampling_size[i] = sampling.get_sample_fundamental(
...         resolution=sampling_resolution[i], point_group=pg432
...     ).size
>>> plt.figure(figsize=(7, 4))
>>> plt.plot(sampling_resolution, sampling_size, "o")
>>> plt.xlabel("Sampling resolution")
>>> plt.ylabel("Sample size")
>>> plt.tight_layout()
```
![v0_6_get_sample_fundamental_strange_size](https://user-images.githubusercontent.com/12139781/117959274-22d87780-b31c-11eb-96bd-875f5bc47008.png)

I haven't looked into the reason for this, but I've used this function quite a lot from v0.5.1, and I didn't experience this behaviour in sample size as a function of resolution.

**Comment by pc494** [comment]
This is off master at https://github.com/pyxem/orix/commit/13ff938d673af777bbd8f45bee50f3cad037e5b7 (or as near as makes no odds)?

**Comment by hakonanes** [comment]
Yeah, that is the latest commit in the branch with which I ran the example above.

**Comment by pc494** [comment]
okay, I'll have a look

**Comment by pc494** [comment]
This dip comes from the sampling of SO3 (and not the fundamental sector bit of code) which helps limits what it could be. Will keep at it,

**Comment by pc494** [comment]
Fixed in #193
